openshift-infra/tower_vars.yml

```diff
@@ -62,3 +61,4 @@ tower_workflow_template: "0-Push_Button_Deploy_OpenShift_on_AWS"
 tower_workflow_template_extra_vars: "tower_workflow_template_extra_vars.yml"
 # Wire up job_templates in this file
 tower_workflow_template_schema_path: "tower_workflow_template_schema.yml"
+tower_workflow_job_launch: true
```

Comment:
This only get executed if you set `tower_config: true`. I can certainly set `tower_workflow_job_launch` to false and anyone who wants can override it to true. The students wouldn't be executing this at all, and the Tower instances the students get would be blank (i.e. `tower_config: false` and thus `tower_workflow_job_launch` would never happen). 

Given this, would you want this defaulted to true or false? The idea for us testing (at least what I was envisioning) is we fire off `aws_lab_launch.yml` with `tower_config: true` and it will configure everything and kick off the workflow. This would be a great way to test 50 or 100 clusters coming up at once, without having to log into each one and kick off the workflow.

Thoughts?